flow.py

```diff
@@ -0,0 +1,122 @@
+class Flow(object):
+    """ A flow class that represents active connections between
+    hosts and routers.
+
+    Args:
+        ns: Instance of the NetworkSimulator class
+        flow_id: Unique id identifying the flow
+
+    Attributes:
+        flow_id: Unique id identifying the flow
+        src: The flow's source node
+        dest: The flow's destination node
+        data_amount: Data capacity of the flow
+
+        window_size: The size of the window
+        unacknowledged_packets: The list of packets with no acknowledgement
+        current_packet: The current packet in the flow
+        num_packets:  The number of packets that have been through the flow
+        last_acknowledged: The last acknowledged packet in the flow
+    """
+    def __init__(self, ns, flow_id):
```

Comment:
Need to pass in other attributes like src, dest, data amount, etc (all the stuff in JSON)